fs/communitarian-discourse-plugin
# [#173960358] Renew resolutions

https://www.pivotaltracker.com/story/show/173960358

## How to test locally
Run both application server and sidekiq.
Create a resolution.
Then visit `localhost:3000/sidekiq/scheduled`, you **should see 2 jobs**: ClosePoll (job from poll plugin) and ReopenResolution (our custom job).
Click on ReopenResolution job and enqueue it, wait for a minute and see that there should be a **new resolution opened** in the same topic and **2 more jobs enqueued**.
